content/docs/how-we-work/baseline/_index.md

```diff
@@ -169,7 +169,9 @@ Since a CI pipeline is considerably more complex, an example describes the proce
 
 As we are on GitHub, we use Dependabot for tracking updates. We typically track updates for all the dependencies we use (typically PHP, JavaScript, and GitHub Actions itself). We schedule alerts on a weekly basis unless there is a specific need. We also [group](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) related dependencies so that we are not flooded by pull requests and associated CI runs.
 
-As before, use the contrib tracker project as an example of [a Dependabot configuration file](https://github.com/contrib-tracker/backend/blob/main/.github/dependabot.yml) and change it to suit your needs.
+As before, use the contrib tracker project as an example of [a Dependabot configuration file](https://github.com/contrib-tracker/backend/blob/main/.github/dependabot.yml) and change it to suit your needs. Make sure to also pull in configuration related checks done in [pipeline](https://github.com/contrib-tracker/backend/blob/main/.github/workflows/ci.yml) that validates if there are any configuration differences due to the update.
```

Comment:
```suggestion
As before, use the contrib tracker project as an example of [a Dependabot configuration file](https://github.com/contrib-tracker/backend/blob/main/.github/dependabot.yml) and change it to suit your needs. Make sure to also pull in configuration related checks done in [pipeline](https://github.com/contrib-tracker/backend/blob/79feee6b0e76a8450e07d2bd0e494ac67fd6c3a4/.github/workflows/ci.yml#L128-L160) that validates if there are any configuration differences due to the update.
```

@Kalaiselvan88, I think a permalink highlighting the lines will be clearer. Any reason we should stick to a generic link to the file?

Reply:
@hussainweb I added the specific file link as the code for handling configuration is available in the CI. FYI it was not obvious for me as well then I remembered as I was involved while we were working. However for a team member who was not involved, I think its important to point to specific file were they can find this,

Reply:
@hussainweb I found it cannot be done in Markdown? https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet#linking-to-markdown

Reply:
@Kalaiselvan88, I don't think I understand. My code suggestion doesn't work? That's a direct permalink to the lines we want to highlight.